code/headers/mlx90640.hpp

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#include <i2c_bus.hpp>
+#include <mlx90640_i2c.hpp>
+#include <mlx90640_processor.hpp>
+
+namespace r2d2::thermal_camera {
+
+    enum class reading_pattern { INTERLEAVED_MODE, CHESS_PATTERN_MODE };
+
+    class mlx90640_c {
+    private:
+        /*
```

Comment:
```suggestion
        /**
```
start with /** end with `*/` please change this in all the documents